gloo/test/multiproc_test.h

```diff
@@ -15,23 +15,34 @@
 #include <chrono>
 #include <string>
 
+#include "base_test.h"
 #include "gloo/common/logging.h"
 #include "gloo/rendezvous/context.h"
 #include "gloo/rendezvous/file_store.h"
-#include "gloo/transport/tcp/device.h"
+#include "gloo/test/base_test.h"
+#include "gloo/test/openssl_utils.h"
+#include "gloo/transport/tcp/tls/device.h"
 
 namespace gloo {
 namespace test {
 
 const int kExitWithIoException = 10;
 const auto kMultiProcTimeout = std::chrono::milliseconds(500);
 
+const std::vector<Transport> kTransportsForMultiProcTest {
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
+    Transport::TCP_TLS,
```

Comment:
Would I be correct if I assume adding this here will make all existing tests run in both TCP and TCP_TLS modes?

Reply:
yes, except not all, but only Transport tests